core/Plugin/Manager.php

```diff
@@ -778,6 +777,15 @@ public function getActivatedPlugins()
         return $this->pluginsToLoad;
     }
 
+    public function getActivatedPluginsFromConfig()
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
+
+        $plugins = $this->sortPluginsSameOrderAsGlobalConfig($plugins);
+
+        return $plugins;
+    }
```

Comment:
@tsteur I have added this method but it kinda duplicates this one just above:

```php
    public function getActivatedPlugins()
    {
        return $this->pluginsToLoad;
    }
```

However replacing the existing method by the one I added just blew up everything… I think it's simply that the existing one has a bad name (it doesn't return the activated plugins, it returns the ones the plugin manager is supposed to load). I could rename it from `getActivatedPlugins()` to `getPluginsToLoad()` if it makes the diff any better.